app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -42,6 +43,10 @@ def participant_id_filter
         filter[:participant_id]
       end
 
+      def updated_since_filter
+        filter[:updated_since]
+      end
```

Comment:
Let's use the new and improved `ApiFilter` concern for it :)

Reply:
@kicferk1 🙏 Thank you!

Reply:
@kicferk1 Much better, thanks. Updated.